app/helpers/application_helper.rb

```diff
@@ -27,6 +27,17 @@ def feature?(feature, user = nil)
     OpenFoodNetwork::FeatureToggle.enabled?(feature, user)
   end
 
+  def language_meta_tags
+    return if I18n.available_locales.one?
+
+    I18n.available_locales.map do |locale|
+      tag.link(
+        hreflang: locale.to_s.gsub("_", "-").downcase,
+        href: "#{request.protocol}#{request.host_with_port}/locales/#{locale}"
```

Comment:
I'm wondering if it would be better to use `ENV["SITE_URL"]` but I'm happy with this solution, too.